src/server/graphql/models/TeamMember/teamMemberMutation.js

```diff
@@ -74,16 +99,20 @@ export default {
           subtype: 'alreadyJoined'
         });
       }
+
+      const usersOnTeam = await r.table('TeamMember').getAll(teamId, {index: 'teamId'}).count();
+
       // add user to TeamMembers
       const newTeamMember = {
+        checkInOrder: usersOnTeam + 1,
```

Comment:
the newest member should go last so they can see how it's done.
after a meeting is ended, the reshuffle will occur (not yet implemented).